### DISCUSSION TITLE
Plugin Required GoToR

### DISCUSSION THREAD

**Discussion by Mano2731995** [farag2/Adobe-Acrobat-Pro-DC-Setup#3] [General]
I'm using Adobe Acrobat Reader DC. I When I trying to click on a (link) ,  a Security Warning pops up that states "this document is trying to access: \users\xyz\desktop\Untitled-1.pdf" If you trust this document, choose Allow. If you do not trust this document, choose Block. I chose Allow. The next pop up states: Acrobat Reader... The plug-in required by this "GoToR" action is not available. Information about the missing plug-in may be available on Adobe's Website.

 

There is no mention to the actual plug-in name required. How do I get this problem fixed?

**Comment by farag2** [comment]
Could you attach a screenshot?

**Reply by Mano2731995** [reply]
![IMG_20221112_104026.jpg](https://user-images.githubusercontent.com/117537492/201458308-974c5982-ba81-464f-a2e1-804ac88ef4d4.jpg)

**Comment by farag2** [comment]
So there is a plug-in missing. Ho2 can I help you regarding my script?)
If you still have any questions, ask them in our Telegram group: https://t.me/sophia_chat
